Autopilot/boardfiles/Src/freertos.c

```diff
@@ -88,19 +89,21 @@
 
 // TODO: Confirm the period at which each of the threads are to be run and update them
 static const int PERIOD_ATTITUDEMANAGER_MS = 100;
+static const int PERIOD_SENSORDATA_MS = 100;
```

Comment:
these periods are wrong. Sensordata should be 200 Hz (5ms) attitude manager should also be 200 Hz and path manager should be 50 Hz, telemetry should also be 50 Hz.